docs/guides/v8-upgrade-guide.md

```diff
@@ -70,6 +94,10 @@ We have removed the following packages:
 **Note:** In case you need them, the 7.x versions are still on npm, but they will not receive further feature updates.
 You can still file bugs, and we will likely fix them or submit pull requests if you want to see a new feature in them.
 
+### TypeScript
```

Comment:
We could put a warning here too about it being WIP, so they can expect some weird warnings for now.

Reply:
done